Sources/Logging/Logging.swift

```diff
@@ -1330,6 +1370,7 @@ private final class WarnOnceBox {
 // MARK: - Sendable support helpers
 
 extension Logger.MetadataValue: Sendable {}
+extension Logger.Storage: @unchecked Sendable {}
```

Comment:
Can we move this conformance to the `Storage` at the again instead of an extension and add a common that it is safe since we manually implement CoW
```suggestion
extension Logger.Storage: @unchecked Sendable {}
```